simpleflow/activity.py

```diff
@@ -37,7 +37,7 @@ def wrap(func):
     return wrap
 
 
-class Activity(object):
+class ActivityType(object):
```

Comment:
define vocab problem?

PS: I was summoned to speak now by mail, so here I appear!